tests/parser/parser.cpp

```diff
@@ -1531,23 +1532,18 @@ TEST(Parser, parseResets)
         "<model xmlns=\"http://www.cellml.org/cellml/2.0#\" id=\"mid\">\n"
         "  <component name=\"component2\" id=\"c2id\">\n"
         "    <variable name=\"variable1\" id=\"vid\"/>\n"
-        "    <reset order=\"1\" id=\"rid\">\n"
-        "      <when order=\"5\">\n"
+        "    <variable name=\"variable2\" id=\"vid2\"/>\n"
+        "    <reset order=\"1\" id=\"rid\" variable=\"variable1\" test_variable=\"variable2\">\n"
+        "      <test_value>\n"
```

Comment:
Everything in this test should have an id, this doesn't.

Reply:
Is that a general thing for cellml stuff everywhere?  I thought ids were optional?

Reply:
Ignore me, just read the name of the test it's in ... :)

Reply:
Done